pattern/text/tree.py

```diff
@@ -1218,7 +1218,7 @@ def xml(string, token=[WORD, POS, CHUNK, PNP, REL, ANCHOR, LEMMA]):
 # Elements:
 XML_TEXT     = "text"     # <text>, corresponds to Text object.
 XML_SENTENCE = "sentence" # <sentence>, corresponds to Sentence object.
-XML_CHINK    = "chink"    # <chink>, where word.chunk.type=None.
+XML_NONCHUNK = "nonchunk" # <nonchunk>, where word.chunk.typNONe=Uone.
```

Comment:
Is it supposed to say "typNONe=Uone." or as the original "word.chunk.type=None" in the original comment?